suites/pacific/cephadm/test-container-cli-args.yaml

```diff
@@ -0,0 +1,373 @@
+              args:
+                node: node1
+                labels: apply-all-labels
+          - config:
+              command: apply_spec
+              service: orch
+              specs:
+                - service_type: host
+                  address: true
+                  labels: apply-all-labels
+                  nodes:
+                    - node2
+                    - node3
+                - service_type: host
+                  address: true
+                  labels: apply-all-labels
+                  nodes:
+                    - node4
+  - test:
+      name: Service deployment with spec
+      desc: Add services using spec file.
+      module: test_cephadm.py
+      polarion-id: CEPH-83574887
+      config:
+        steps:
+          - config:
+              command: apply_spec
+              service: orch
+              validate-spec-services: true
+              specs:
+                - service_type: mon
+                  placement:
+                    nodes:
+                      - node1
+                      - node2
+                      - node3
+                  extra_container_args:
+                    - "--cpus=2"
+                - service_type: mgr
+                  placement:
+                    label: mgr
+                  extra_container_args:
+                    - "--cpus=2"
+                - service_type: prometheus
+                  placement:
+                    count: 1
+                    nodes:
+                      - node1
+                  extra_container_args:
+                    - "--cpus=1"
+                - service_type: grafana
+                  placement:
+                    nodes:
+                      - node1
+                  extra_container_args:
+                    - "--cpus=1"
+                - service_type: alertmanager
+                  placement:
+                    count: 2
+                    label: alertmanager
+                  extra_container_args:
+                    - "--cpus=1"
+                - service_type: node-exporter
+                  placement:
+                    host_pattern: "*"
+                  extra_container_args:
+                    - "--cpus=1"
+                - service_type: crash
+                  placement:
+                    host_pattern: "*"
+                  extra_container_args:
+                    - "--cpus=1"
+                - service_type: osd
+                  service_id: all-available-devices
+                  encrypted: "true"                     # boolean as string
+                  placement:
+                    host_pattern: "*"
+                  data_devices:
+                    all: "true"                         # boolean as string
+                  extra_container_args:
+                    - "--cpus=2"
+          - config:
+              command: shell
+              args:                 # sleep to get all services deployed
+                - sleep
+                - "300"
+  - test:
```

Comment:
since we have one polarion test case for service deployment with container_args, can we have one test case here also?
steps:
    - mon
    - mgr
    - osd
    ..
    ..
    - SNMP

Reply:
Done.